build.sc

```diff
@@ -195,9 +194,14 @@ object akka extends Module {
 }
 
 object examples extends Module {
+  trait ExamplesJvmCommon extends Bot4sTelegramCrossPlatform {
+    override def ivyDeps = super.ivyDeps() ++ Agg(
```

Comment:
We could place here one more common part:
```scala
override val platformSegment: String = "jvm"
```

Reply:
Good catch, you can move that in the `ExamplesJvmCommon` trait indeed :ok_hand: